internal/receiver/databricksreceiver/metadata.yaml

```diff
@@ -215,181 +221,212 @@ metrics:
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.minor_gc.count:
     enabled: true
-    description: n/a
+    description: Total minor GC count. For example, the garbage collector is one of Copy, PS Scavenge, ParNew, G1 Young Generation and so on.
+    unit: "{gc}"
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.minor_gc.time:
     enabled: true
-    description: n/a
+    description: Elapsed total minor GC time.
+    unit: ms
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.off_heap.execution.memory:
     enabled: true
-    description: n/a
+    description: Peak off heap execution memory in use, in bytes.
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.off_heap.storage.memory:
     enabled: true
-    description: n/a
+    description: Peak off heap storage memory in use, in bytes.
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.off_heap.unified.memory:
     enabled: true
-    description: n/a
+    description: Peak off heap memory (execution and storage).
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.on_heap.execution.memory:
     enabled: true
-    description: n/a
+    description: Peak on heap memory (execution and storage).
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.on_heap.storage.memory:
     enabled: true
-    description: n/a
+    description: Peak on heap storage memory in use, in bytes.
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.on_heap.unified.memory:
     enabled: true
-    description: n/a
+    description: Peak on heap memory (execution and storage).
+    unit: By
     gauge:
       value_type: double
     attributes:
       [cluster.id, spark.app.id, pipeline.id, pipeline.name]
   databricks.spark.executor_metrics.process_tree.jvm_rss.memory:
     enabled: true
-    description: n/a
+    description: "Resident Set Size: number of pages the process has in real memory. This is just the pages which count toward text, data, or stack space. This does not include pages which have not been demand-loaded in, or which are swapped out. Enabled if spark.executor.processTreeMetrics.enabled is true."
+    unit: "{pages}"
```

Comment:
interesting they don't get the bytes and would require you to know the page size.

Reply:
maybe I interpreted the docs wrong, fwiw.